executor/inspection_summary.go

```diff
@@ -366,17 +357,16 @@ var inspectionSummaryRules = map[string][]string{
 		"pd_grpc_completed_commands_rate",
 		"pd_handle_request_duration",
 		"pd_handle_request_ops",
-		"pd_handle_requests_duration_avg",
-		"pd_handle_requests_duration",
+		"pd_handle_request_duration_avg",
 		"pd_handle_transactions_duration",
 		"pd_handle_transactions_rate",
 		"pd_hotspot_status",
 		"pd_label_distribution",
 		"pd_operator_finish_duration",
 		"pd_operator_step_finish_duration",
-		"pd_peer_round_trip_time_seconds",
+		"pd_peer_round_trip_duration",
```

Comment:
Why change this?

Reply:
Since I change the table name `pd_peer_round_trip_time_seconds` to `pd_peer_round_trip_duration`.

Just keep unified, mostly time cost table name end with `duration` .